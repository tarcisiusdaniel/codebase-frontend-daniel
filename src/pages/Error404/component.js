import React from 'react';
import { Link } from 'react-router-dom';
import Typography from '@material-ui/core/Typography';
import { ROUTES } from '../../configs';

export default class Component extends React.Component {
  render() {
    return (
      <section>
        <Typography variant="display2">Page not found</Typography>
        <Typography>
          <Link to={ROUTES.HOME()}>Back to home</Link>
        </Typography>
      </section>
    );
  }
}