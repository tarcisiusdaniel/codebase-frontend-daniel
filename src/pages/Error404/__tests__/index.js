import React from 'react';
import renderer from 'react-test-renderer';
import Error404 from '../index';

jest.mock('react-router-dom');

describe('Error404', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Error404 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});