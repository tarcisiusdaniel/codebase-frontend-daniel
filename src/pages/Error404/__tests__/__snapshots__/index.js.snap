// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 renders correctly 1`] = `
<section>
  <h1
    className="MuiTypography-root-1 MuiTypography-display2-4"
  >
    Page not found
  </h1>
  <p
    className="MuiTypography-root-1 MuiTypography-body1-10"
  />
</section>
`;
